### ISSUE TITLE
DE German translation

### ISSUE THREAD

**Issue by PeterNerlich** [ncase/crowds#13]
I wanna have a German version of this! Not sure how soon I can deliver that though.

If anyone wants that, too, have a look whether I started (and when my last change was) [over at my fork](https://github.com/PeterNerlich/crowds). Please also feel free to ask me about this or take it out of my hands if I don't continue in a reasonable time, too.

**Comment by brasilikum** [comment]
https://github.com/ncase/crowds/pull/18

Maybe let's consolidate

**Comment by silberzwiebel** [comment]
What's the status of your two WIP translations? How could I best help without doing work twice? As far as I can tell, both of you have started the translation already?

**Comment by brasilikum** [comment]
Yes, but @PeterNerlich is further. I suggest you work on his fork

**Comment by silberzwiebel** [comment]
Alright, I just started to add up to @PeterNerlich's translation at [my fork](https://github.com/silberzwiebel/crowds/tree/translation-de). It might be useful to do WIP pull request all over at some central place (here?)? So anyone can pick up the current status relatively easily?
Anyway, I have no background in sociology and wondered how to best translate "contagion"? Is there a proper term in German sociology? Ansteckung? Übertragung?

**Comment by PeterNerlich** [comment]
If not with origin, feel free to make WIP PRs to my fork. This should also remind me of it more often and subconsciously raise my perceived urgency, so I get back to it sooner...

**Comment by silberzwiebel** [comment]
See https://github.com/PeterNerlich/crowds/pull/1.
